openmsupply/conforma-server
# #1303F - Part1 & Part2 - Return reviewer list action when changes required

Front-end PR: https://github.com/openmsupply/conforma-web-app/pull/1311

Testing: Please use the latest snapshot in our shared folder: https://drive.google.com/drive/folders/1pAhRoqqVNrWggEYEtlZ8T5ae2cWU_MDP 

Please, remember to run `yarn migrate` on the snapshot you are working with to get the changes applied. 

Testing:
- Login as `jess` and see there is already an application for _Cadastramento de entidade_ to review
- Self assign => Check that in the applications list the action showing "Start review"
- Start the review, add one 'Non conform' then submit as _Request further information_ (LOQ) => In the list the action now is "Awaiting response" 

Basically changes database for:
- Fixes front-end as requested by Jess to allow Reviewer to see "action" when the application has been submitted to Applicant for LOQ. 
- Add migration code for version 0.4.4 (to be release) - @CarlosNZ I wasn't sure if that's how this is done - for the next release
- Update core_templates with new Pt strings & after running the migration code
